projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -526,6 +540,8 @@ public function update_translations() {
 	 * Get translations.
 	 *
 	 * @param array $translation - the translation.
+	 *
+	 * @return string
```

Comment:
Looks like this is actually a weirdly named function for use with `array_filter()`. Someday we might refactor it to be an inline anonymous function, but for now
```suggestion
	 * Test whether the translation matches `$this->slug`.
	 *
	 * @param array $translation - the translation.
	 *
	 * @return bool
```